.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+Closes #???
+
+The following tasks have been completed:
+
+- [ ] Modified Web platform tests (link)
+
+Implementation commitment:
+
+- [ ] WebKit (link to issue)
+- [ ] Chromium (link to issue)
+- [ ] Gecko (link to issue)
+
+Documentation and checks
+
+- [ ] Affects privacy
+- [ ] Affects security
+- [ ] Pinged MDN
```

Comment:
do we ever actually require this?

Reply:
Not sure. I stole from DC API, which stole it from CredMan :)

Reply:
If we're keeping it, is there a link we can add instructing where/how to do this?

Reply:
removed